azure-pipelines.yml

```diff
@@ -217,23 +217,13 @@ jobs:
     dependsOn:
       - da_ghc_lib
       - check_for_release
```

Comment:
```suggestion
```

Reply:
I’ll fix it afterwards so we can get the snapshot in.

Reply:
Eh, you're going to have to re-trigger Windows anyway, so feel free to include it here.

Reply:
:sob: